src/SDKs/Batch/DataPlane/Azure.Batch/Azure.Batch.csproj

```diff
@@ -0,0 +1,66 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>Azure.Batch</PackageId>
+    <Description>This client library provides access to the Microsoft Azure Batch service.</Description>
+    <VersionPrefix>6.1.0</VersionPrefix>
+    <DefineConstants>$(DefineConstants);CODESIGN</DefineConstants>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <AssemblyName>Microsoft.Azure.Batch</AssemblyName>
+    <PackageTags>Microsoft;Azure;Batch;windowsazureofficial</PackageTags>
+    <DocumentationFile>bin\$(Configuration)\$(TargetFramework)\$(AssemblyName).xml</DocumentationFile>
+    <SignAssembly>true</SignAssembly>
+    <DelaySign>true</DelaySign>
+    <AssemblyOriginatorKeyFile>$(LibraryToolsFolder)\MSSharedLibKey.snk</AssemblyOriginatorKeyFile>
+    <RestorePackagesPath>$(LibraryNugetPackageFolder)</RestorePackagesPath>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
+    <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
+    <PackageReleaseNotes>For detailed release notes, see: https://github.com/Azure/azure-sdk-for-net/blob/vs17Dev/src/SDKs/Batch/DataPlane/changelog.md</PackageReleaseNotes>
+  </PropertyGroup>
+
+  <PropertyGroup Condition=" '$(TargetFramework)'=='net452' ">
+    <DefineConstants>$(DefineConstants);FullNetFx</DefineConstants>
+  </PropertyGroup>
+
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'netstandard1.4' ">
+    <!-- Portable is defined for legacy reasons, if updating AutoRest removes it from BatchErrorException we can remove it here -->
+    <DefineConstants>$(DefineConstants);netstandard14;PORTABLE</DefineConstants>
```

Comment:
@matthchr portable is no longer a library type we support, so adding a constant that says this code is Portable is misleading.
Rather either your code targets full desktop for that we have used the constant as FullNetFx (so the idea is this can be any version .NET 4.5.2 or .NET 4.6 or !FullNetFx (which is essentially .NET Core/.NET Standard